include/cpuinfo.h

```diff
@@ -2042,6 +2043,18 @@ static inline bool cpuinfo_has_arm_sve2(void) {
 #endif
 }
 
+// Function to get the max SVE vector length on ARM CPU's which support SVE.
+static inline int cpuinfo_get_max_arm_sve_length(void) {
+#if CPUINFO_ARCH_ARM64
+	if ((cpuinfo_isa.sve || cpuinfo_isa.sve2) && (cpuinfo_isa.svelen > 0))
+		return cpuinfo_isa.svelen;
+	else
+		return -1; // Returns -1 on Non SVE supported ARM CPU's.
+#else
+	return -1; // Returns -1 on Non ARM CPU's.
+#endif
+}
```

Comment:
Why not keep it simple? One does not need a check, on whether or not svelen is supported, as it should be initialized to 0, shouldn't it?
```suggestion
static inline uint32_t cpuinfo_get_max_arm_sve_length(void) {
#if CPUINFO_ARCH_ARM64
	return cpuinfo_isa.svelen;
#else
	return 0;
#endif
}
```

Reply:
Agree, changes done.